lib/asset.js

```diff
@@ -13,25 +13,36 @@
 'use strict';
 const path = require('path');
 
-class Source {
-    constructor(params, directory="", name="") {
-        this.settings = params;
-
+class Asset {
```

Comment:
+1  we just ned to make sure it doesn't break anything for workers. Looking at the small exposed [api](https://github.com/adobe/asset-compute-sdk#source) it should be ok, the class name is not visible to workers, but worth checking twice.

Reply:
We are not aware of custom workers already being used in production? THose are the only ones I can think which may drill down into the SDK and use this private API. If we don't have knowledge of any custom worker in prod, we can trust the IT tests to tell us if core workers are OK or not.

Reply:
Yes, there are many (even if most of them are test projects by consultants, they all are production for us).

Reply:
It's not. public API so it should not be a breaking change... But would you prefer to release as breaking? Is a strategy defined how we would handle such things?
(Note that I would expect our consultants to possibly drill down into the SDK to use our classes directly if they would need to. This is also why I was hesitating to do class renamings.).

Reply:
Not release as breaking. We must never get into a situation that would require a breaking change. Just make sure it's none. If it is, change it so that it is backwards compatible.

Reply:
I have a memory of this affect meahana for some reason...like if the uploaded rendition was uploaded to cloud without a contenttype, it was displayed weirdly in the meahana ui.

calling download without content-type should not make a difference, AFAICS

Reply:
if we think its too risky to change the name to Asset, maybe we wait and do this in a different PR.

I mean, worst-case scenario some customer uses the `Source` class for their intermediate assets inside their custom worker or something...seems like a long shot because they would really have to dig through the sdk code to get this, but who knows